### PR TITLE
Add MobieStepper example

### PR DESCRIPTION
**Pull request by daileytj**
#### Changes proposed in this Pull Request:
- Add MobileStepper example

### CODE REVIEW

File: router/navigation-drawer.tsx [hunk 1]
```diff
@@ -103,8 +103,8 @@ export const navGroupItems2: NavItem[] = [
         itemID: 'g2i3',
     },
     {
-        subtitle: 'You cant see me, fix InfoListItem',
-        title: 'Alerts',
+        subtitle: 'Click me to clear',
+        title: 'Clear Selection',
         itemID: 'g2i4',
```
Comment (joebochill) [inline router/navigation-drawer.tsx:108]: We probably don't need to keep this in here now.

**Review by joebochill — COMMENTED** [review]
You may want to consider starting to break the App component into multiple files. You could break the stepper out into it's own file (managing it's own state) and then use it in the App.tsx as `<StepperExample/>`. Just a thought.

File: router/navigation-drawer.tsx (hunk 1 above)
Comment (daileytj) [reply router/navigation-drawer.tsx:108]: I was just leaving it in there for regression testing of the parent font weight stuff. I can remove it though. Do you want the entire item removed, or just remove the functionality and change the title to something generic?

Comment (joebochill) [reply router/navigation-drawer.tsx:108]: If it were me, I would set it back to what it was before. And then if we want to have an item to test the styles, make a new item as the last item in the drawer body.

**Review by joebochill — APPROVED** [review]

**Pull request merged**